lib/engine/game/g_18_norway/game.rb

```diff
@@ -453,20 +453,19 @@ def for_graph_city_tokened_by?(city, entity, graph)
           harbor_token?(city, entity)
         end
 
-        def connected?(a, b, corporation)
+        def connected?(a, b, corporation, train)
```

Comment:
"train" might not be a good name of this parameter, as it can be a ship.
How about train_or_ship? Or some other name that includes both? (moveable?)

Reply:
I think that naming is fine. The ships are being implemented as Train objects, so it will be a train that is passed as a parameter.